utilities/scripts/xmlPagsToEntityPags.groovy

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
@jameswennmacher et al. -- what do you think about moving utilities/ under bootstrap/ so we're not adding a new top-level folder to the repo for a single file?